src/unix/mod.rs

```diff
@@ -228,6 +228,11 @@ cfg_if! {
         #[link(name = "root")]
         #[link(name = "network")]
         extern {}
+    } else if #[cfg(target_os = "fuchsia")] {
+        #[link(name = "c")]
+        #[link(name = "mxio")]
+        #[link(name = "unwind")]
```

Comment:
This doesn't seem right to me, but I haven't been able to figure out a better way to get it to work. I tried adding it as `#cfg[attr(..., link(...))]` at the end of https://github.com/rust-lang/rust/blob/master/src/libunwind/libunwind.rs , but that doesn't seem to show up in the final link line.